Translatathon pages (#13444)
ethereum/ethereum-org-website: public/content/translatathon/index.md

---
title: Translatathon hub
lang: en
template: translatathon
---

<CardContainer>
  <EmojiCard
    emoji=':globe_with_meridians:'
    title='Translate ethereum.org'
    description='Translate ethereum.org content into your language and help make the website more accessible'
  />
  <EmojiCard
    emoji=':trophy:'
    title='Win prizes'
    description='Compete for your share of 30,000$ in prizes and opportunities to win Devcon tickets'
  />
  <EmojiCard
    emoji=':bulb:'
    title='Learn about Ethereum'
    description='Learn about different Ethereum related topics while contributing and competing'
  />
</CardContainer>

> 

Reviewer: Looking into this now

> 

Reviewer: Pushed a commit changing the layout styes for this `CardContainer` component. Should fix this issue.

    In order to participate, you need to be older than 18 years and fluent in at least one language in addition to English.
  </CardContent>
  <CardContent>
  <h3>Do I need to be a translator?</h3>
    No. You simply have to be bilingual and suggest human translations (using machine translation is forbidden!) to the best of your ability, no professional experience required.
  </CardContent>
</TwoColumnContent>

<TwoColumnContent>
  <CardContent>
    <h3>How much time do I have to commit?</h3>
    As much as you want. The minimum threshold to be eligible for participatory prizes is 100 translated words, which takes about 10-20 minutes to complete, while competing for the top prizes will require a larger time commitment.
  </CardContent>
  <CardContent>
    <h3>Do I need to be familiar with Ethereum?</h3>
    No. While being familiar with Ethereum can help with your productivity and quality, the Translatathon is as much a learning experience as it is a competition, and everyone is invited to join and learn more about Ethereum while participating.
  </CardContent>
</TwoColumnContent>

<TwoColumnContent>
  <CardContent>
    <h3>What do I need to participate?</h3>

>     We recommend using a computer to translate since using Crowdin will be optimal on desktop. You’ll also need an internet connection and you’re good to go.

Reviewer: ```suggestion
    We recommend using a computer to translate since our translation platform, Crowdin, is optimized for desktop.
```
